{"status": "resolved", "priority": "bug", "title": "mysql warning during Person.create_dummy", "milestone": "", "superceder": "", "nosylist": ["paulproteus, zathras"], "assigned": "", "waitingon": "", "keywords": ["transparency"], "id": "140", "files": [{"url": "http://openhatch.org/bugs/file72/my.cnf", "author": "zathras"}], "history": [{"message": "I fixed this in origin/master and pushed the commit. Zathras, please do a \"git\npull\" or equivalent.\n\nIf you want read what the problem was, run:\n\n$ git show 1e3afc271468c9843a19a49ee702ad81c8af8d4d\n   \n", "author": "paulproteus"}, {"message": "in previous install a dummy person was created but the error was there. Now the\nerror occurs but no dummy is created.\n   \n", "author": "zathras"}, {"message": "sorry, issue is still there.\n2010-09-25 07:48:59,539 refresh_one_launchpad_bug:194 INFO     Refreshing\n<a href=\"https://bugs.launchpad.net/bugs/634308.\">https://bugs.launchpad.net/bugs/634308.</a>\n _mysql_exceptions.Warning: Data truncated for column 'title' at row 1\n   \n", "author": "zathras"}, {"message": "Great to hear!\n   \n", "author": "paulproteus"}, {"message": "on a fresh install this no longer happens\n   \n", "author": "zathras"}, {"message": "Hi! I will try to look into this soon.\n   \n", "author": "paulproteus"}, {"message": "_mysql_exceptions.Warning: Incorrect string value: '\\xC4\\xB1temo...' for column\n'submitter_realname' at row 1\nDB was filled beyond the first session. But error is similar so it seems\nlast line before error: 2010-09-16 21:07:42,649 refresh_one_launchpad_bug:194\nINFO     Refreshing <a href=\"https://bugs.launchpad.net/bugs/593089.\">https://bugs.launchpad.net/bugs/593089.</a>\n   \n", "author": "zathras"}]}